components/builder-core/src/package_graph.rs

```diff
@@ -88,19 +88,23 @@ impl PackageGraph {
         }
     }
 
-    pub fn build<T>(&mut self, packages: T) -> (usize, usize)
+    pub fn build<T>(&mut self, packages: T, feat: Option<()>) -> (usize, usize)
```

Comment:
"feat" could just be a bool instead of Option<()>.. eg have something like "use_build_deps : bool"